src/test/java/nextstep/subway/AcceptanceTest.java

```diff
@@ -1,23 +1,65 @@
 package nextstep.subway;
 
-import io.restassured.RestAssured;
-import nextstep.subway.utils.DatabaseCleanup;
 import org.junit.jupiter.api.BeforeEach;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.boot.test.context.SpringBootTest;
 import org.springframework.boot.web.server.LocalServerPort;
+import org.springframework.http.MediaType;
+
+import io.restassured.RestAssured;
+import io.restassured.response.ExtractableResponse;
+import io.restassured.response.Response;
+import nextstep.subway.utils.DatabaseCleanup;
 
 @SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
-public class AcceptanceTest {
-    @LocalServerPort
-    int port;
-
-    @Autowired
-    private DatabaseCleanup databaseCleanup;
-
-    @BeforeEach
-    public void setUp() {
-        RestAssured.port = port;
-        databaseCleanup.execute();
-    }
+public class AcceptanceTest<T>{
+	@LocalServerPort
+	int port;
+
+	@Autowired
+	private DatabaseCleanup databaseCleanup;
+
+	@BeforeEach
+	public void setUp() {
+		RestAssured.port = port;
+		databaseCleanup.execute();
+	}
+
+	protected ExtractableResponse<Response> post(T request, String uri) {
```

Comment:
```suggestion
	protected <T> ExtractableResponse<Response> post(T request, String uri) {
```
클래스가 아닌 메서드 시그니처에 제네릭을 표시해주는 방법도 있습니다.